app/routes/wow/upload-timers/index.tsx

```diff
@@ -48,19 +48,23 @@ export default Index
 
 const columnList: Array<ColumnList<UploadTimersItem>> = [
   { columnId: 'dataSetID', header: 'Data Set ID' },
-  { columnId: 'dataSetName', header: 'Data Set Name' },
   { columnId: 'lastUploadMinute', header: 'Last Upload Minute' },
   { columnId: 'lastUploadTimeRaw', header: 'Last Upload Time Raw' },
-  { columnId: 'lastUploadUnix', header: 'Last Upload Unix' },
   { columnId: 'region', header: 'Region' },
-  { columnId: 'tableName', header: 'Table Name' }
+  {
+    columnId: 'dataSetName',
+    header: 'Data Set Name',
+    accessor: ({ row }) => (
+      <p className="text-start">{row.dataSetName.join(', ')}</p>
```

Comment:
try adding the className `py-2 px-3 w-[200px] overflow-x-scroll`, this should give you more managable table cell widths